jcl/src/openj9.dtfjview/share/classes/com/ibm/jvm/dtfjview/commands/OpenCommand.java

```diff
@@ -119,7 +119,7 @@ private void imagesFromCommandLine(String args[]) {
 				File file2 = new File(args[1]);
 				images[0] = factory.getImage(file1, file2);
 			}
-			logger.fine(String.format("Time taken to load image %d ms", System.currentTimeMillis() - current));
+			logger.fine(String.format("Time taken to load image %d ms", System.nanoTime() - current));
```

Comment:
The message indicates "ms", but now it's "ns".

Reply:
Updated.